src/modules/search.js

```diff
@@ -25,8 +25,11 @@ function createSearchUrl(query, parameters, userParameters, options, isVoiceSear
   queryParams.i = clientId;
   queryParams.s = sessionId;
 
-  // Validate query (term) is provided
-  if (!query || typeof query !== 'string') {
+  // Trim non breaking spaces from query
+  const queryTrimmed = helpers.trimNonBreakingSpaces(query);
+
+  // Validate query (term) is provided and consists of more than non-breaking spaces
+  if (!queryTrimmed || typeof queryTrimmed !== 'string') {
```

Comment:
This was added to ensure we don't dispatch requests consisting of solely non-breaking spaces (ex:    )